infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationEvent.java

```diff
@@ -57,7 +57,9 @@ public enum LogReplicationEventType {
      * @param type log replication event type
      */
     public LogReplicationEvent(LogReplicationEventType type) {
-        this(type, LogReplicationEventMetadata.empty());
+        this.type = type;
+        this.eventId = Utils.genPseudorandomUUID();
+        this.metadata = new LogReplicationEventMetadata(eventId);
```

Comment:
this would mean that the request id and event id are the same.  They must be different, right?

Reply:
ok.  It is more confusing when looking at other constructors (one of them takes the metadata and generates a random event id).  A question about why event and request ids are same in this case and different in others is natural to occur.  Even though there is no functional relation between them, the code shows otherwise.

Is it possible to change this new behavior or clean up the existing constructors so that the purpose/role of each id is clear?

Reply:
Sure, I see your point...I'll review to see how I can keep the constructor's behaviour consistent. Thanks!